src/NServiceBus.Core/Encryption/ConfigureRijndaelEncryptionService.cs

```diff
@@ -107,25 +107,42 @@ internal static void VerifyKeys(List<string> expiredKeys)
             }
         }
 
-        static IEncryptionService BuildRijndaelEncryptionService(string encryptionKey,List<string> expiredKeys)
+        static IEncryptionService BuildRijndaelEncryptionService(string encryptionKey, List<string> expiredKeys)
         {
             return new RijndaelEncryptionService(encryptionKey, expiredKeys);
         }
 
         /// <summary>
-        /// Register a custom <see cref="IEncryptionService"/> to be used for message encryption.
+        /// Register a custom <see cref="IEncryptionService" /> to be used for message encryption.
         /// </summary>
-        /// <param name="config">The <see cref="BusConfiguration"/> instance to apply the settings to.</param>
-        /// <param name="func">A delegate that constructs the insatnce of <see cref="IEncryptionService"/> to use for all encryption.</param>
+        /// <param name="config">The <see cref="BusConfiguration" /> instance to apply the settings to.</param>
+        /// <param name="func">
+        /// A delegate that constructs the instance of <see cref="IEncryptionService" /> to use for all
+        /// encryption.
+        /// </param>
+        [ObsoleteEx(ReplacementTypeOrMember = "RegisterEncryptionService(this BusConfiguration config, Func<IEncryptionService> func)", RemoveInVersion = "7.0", TreatAsErrorFromVersion = "6.0", Message = "It is no longer possible to access the builder to create an encryption service. If you require container access use your container directly in the factory.")]
```

Comment:
we dont need xml doco for obsoletes. since effectively it now doesnt do what the doco says